spec.html

```diff
@@ -936,6 +989,30 @@ <h1>The AsyncContext Object</h1>
 
     <emu-clause id="sec-asynccontext-abstract-operations">
       <h1>AsyncContext Abstract Operations</h1>
+
+      <emu-clause id="sec-hostgettoplevelasynccontextmapping" type="abstract operation">
+        <h1>
+          HostGetTopLevelAsyncContextMapping (
+            _module_: a Module Record or *null*
+          ): a List of Async Context Mapping Records</h1>
+        <dl class="header">
+          <dt>description</dt>
+          <dd>It allows hosts to provide the top level async context mapping.</dd>
+        </dl>
+        <p>An implementation of HostGetTopLevelAsyncContextMapping must conform to the following requirements:</p>
+        <ul>
+          <li>It must return a List of Async Context Mapping Records.</li>
+          <li>The result List must contain the same Async Context Mapping Records for a given module record.</li>
```

Comment:
FWIW, [HostHasSourceTextAvailable](https://262.ecma-international.org/15.0/#sec-hosthassourcetextavailable) has a similar requirement worded as "It must be deterministic with respect to its parameters. Each time it is called with a specific _func_ as its argument, it must return the same result".  It might be worth rephrasing to be consistent with that language.